package.json

```diff
@@ -21,13 +21,17 @@
     "@mdx-js/react": "^1.6.22",
     "@smallstep/internal": "./vendor/internal",
     "@smallstep/step-ui": "./vendor/step-ui",
+    "@styled-icons/fa-solid": "^10.34.0",
+    "algoliasearch": "^4.13.1",
     "cross-fetch": "^3.1.5",
     "formik": "^2.2.9",
     "gatsby": "^3.12.1",
+    "gatsby-plugin-algolia": "^0.26.0",
     "gatsby-plugin-compile-es6-packages": "^2.1.1",
     "gatsby-plugin-mdx": "^2.12.0",
     "gatsby-plugin-next-seo": "^1.8.0",
     "gatsby-plugin-remove-trailing-slashes": "^3.14.0",
+    "gatsby-plugin-styled-components": "^5.14.0",
```

Comment:
This dependency will end up going away with the port to material-ui and step-ui as well.